var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -410,7 +416,8 @@ def define_tpl_enable(cmake_var, spec_var=None):
 
         options.extend([
             define('Trilinos_VERBOSE_CONFIGURE', False),
-            define('Trilinos_ENABLE_TESTS', False),
+            define('Trilinos_ENABLE_TESTS', (
+                True if self.run_tests else False)),
```

Comment:
```suggestion
                bool(self.run_tests)),
```

Reply:
#19119 I would like to see run_tests actually do something for Trilinos, but if that's outside the scope of your PR, that's okay.